public/apps/configuration/utils/combo-box-utils.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ *   Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+import { EuiComboBoxOptionOption } from '@elastic/eui';
+import { Dispatch, SetStateAction } from 'react';
+import { StringRepresentable, curry } from 'lodash';
+import { appendElementToArray } from './array-state-utils';
+
+// Build option objects for EuiComboBox
+export function stringToComboBoxOption(option: string): EuiComboBoxOptionOption {
+  return { label: option };
+}
+
+// Unbuild EuiComboBox option objects to string[]
+export function comboBoxOptionToString(option: EuiComboBoxOptionOption): string {
```

Comment:
Same here. I think you are handling singular case, not plural case. 
Unbuild an EuiComboBox option object to string

Reply:
Done improvement.